client/src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import Authorize from './components/authorize';
+//import Authorize from './components/authorize';
```

Comment:
Need to remove all the commented out code

Reply:
I have removed all commented out code I could find and will push.  Nice find!!